types/nodemark/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "esModuleInterop": true,
+        "module": "commonjs",
```

Comment:
```suggestion
        "module": "commonjs",
```
this is default (or set by user), when consuming packages, here not required, see:
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

Reply:
agree, adopted the suggestion